client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -7,14 +7,19 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    '&&': {
+      fontSize: 13,
```

Comment:
Alternatively, we can assign a font size to different variants within MUI's theme and use those variants throughout the app to ensure a consistent font-size

Ditto where applicable